themes/src/main/resources/theme/keycloak-preview/account/resources/content.json

```diff
@@ -56,5 +56,14 @@
     "modulePath": "/content/my-resources-page/MyResourcesPage.js",
     "componentName": "MyResourcesPage",
     "hidden": "!features.isMyResourcesEnabled"
+  },
+  {
+    "id": "deleteAccount",
```

Comment:
I think we should add some icon for this page. It's probably the only page (present as a box on the Welcome Screen) without one.

Reply:
maybe the `fa-trash` or `pf-icon-delete` ? 

https://www.patternfly.org/v4/design-guidelines/styles/icons

Reply:
I used `pf-icon-disconnected`